app/views/crops/show.html.erb

```diff
@@ -48,7 +48,10 @@
   </div>
   <div class="guide-results-container small-6 columns">
     <h2 class="text-center"><%= @guides.empty? ? "There are no guides for this crop yet" : "Guides for this crop" %></h2>
-    <p class="text-center"><%= "Please consider creating one if you can!" if @guides.empty? %></p>
+
+    <% if @guides.empty? %>
+      <p class="text-center">Please consider <%= link_to("creating one", new_guide_path) %> if you can!</p>
+    <% end %>
```

Comment:
I think this might be good enough.

It's only one level deep and only tests one condition. Shouldn't be hard to maintain later on down the road. :+1: